charts/opentelemetry-kube-stack/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-kube-stack
-version: 0.2.3
+version: 0.2.4
```

Comment:
we can change this to 0.3.0

Reply:
Definitely, changed in https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1384/commits/d6c39c1e7107751d2dc581226b6ea7ac57166db4